apps/freelance/src/redux/profileQuestions/profileQuestions1Api.ts

```diff
@@ -0,0 +1,27 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { profileQ1Payload } from 'redux/services/types/profileQuestions1.types';
+
+declare let process: {
+  env: {
+    NX_API_URL: string;
+  };
+};
+const baseUrl: string = (process.env.NX_API_URL as string) || '';
```

Comment:
move it to a constant file. in other case you will have a lot of same duplications

Reply:
fixed